source/include/TypeDefiner.h

```diff
@@ -105,6 +105,9 @@ class TypeDefiner
 
 #else
 
+    // With non-unique names(AKA overloaded) only the C entry points are appended.
+    #define DEFINE_VIREO_FUNCTION_TYPED(_root_, _type_, _proto_)  DEFINE_VIREO_FUNCTION_CUSTOM(_root_, _root_##_type_, _proto_)
```

Comment:
Is there something that can be done to avoid defining these functions in two places?